data/strings/strings.txt

```diff
@@ -10652,6 +10888,7 @@
     ja = 中間地点を調整してください。
     ko = 중간 지점을 조정하세요.
     lt = Suderinkite tarpinį tašką.
+    lv = Precizējiet starppunktu.
```

Comment:
"Pielabojiet starppunktu"?

Reply:
Then, I think, it stylistically would be better to write "Labojiet starppunktu." without the prefix.
But I'm not sure at what circumstances this message shows up?

Reply:
Navigation cannot build route with the chosen "intermediate point" and asks to fix the route by moving it somewhere else. "Please adjust your intermediate point." isn't the best text that can be written in English either. @biodranik @rtsisyk ideas?

![photo_5436154095711940181_y](https://github.com/user-attachments/assets/5f5a503f-70c6-45f2-9111-40c081e3afda)

Reply:
It is a pretty generic comment text, so it functions to encourage the user to do something without specifying the problem. I would guess  that "Labojiet starppunktu" or "Pārbaudiet starppunktu" both work.